Specifications/Compiler/com.apple.compilers.llvm.clang.1_0.analyzer.xcspec

```diff
@@ -0,0 +1,27 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Compiler;
+    Identifier = com.apple.compilers.llvm.clang.1_0.analyzer;
+    BasedOn = com.apple.compilers.llvm.clang.1_0;
+    Name = "Static Analyzer";
+
+    RuleName = "Analyze $(InputFile)";
+
+    InputFileTypes = (
+        "sourcecode.c.c",
+        "sourcecode.c.objc",
+        "sourcecode.cpp.cpp",
+        "sourcecode.cpp.objcpp",
+    );
+
+    OutputDir = "$(TEMP_DIR)/StaticAnalyzer";
```

Comment:
You sure this is right? Since this specification is definitely incomplete, I might prefer removing all of the configuration past `Name` and just adding a `/* TODO: Everything. */` kind of comment.